time_service.rb

```diff
@@ -0,0 +1,42 @@
+require 'rack'
+require 'hoodoo'
+
+
+#This is the implementation class. 
+#the implementation class is refered to by the interface class therefore, Implementation class must be defined first.
+#naming convention: FooImplementation.
+class TimeImplementation < Hoodoo::Services::Implementation
+    def show( context )
+      context.response.set_resource( { 'time' => Time.now.utc.iso8601 } )
+    end
+end
+
+#This is the interface class.
+#The interface class takes the name of the resource and a block which describes the resource.
+class TimeInterface < Hoodoo::Services::Interface
+    interface :Time do #name of resource as a symbol.
+    endpoint :time, TimeImplementation #define the endpoint of the resource using uri fragment time followed by the implementation class name. interface version is not specified. URI defaults to path of .../v1/time
+    public_actions :show #public_actions calls the show method found in the TimeImplementation class. 
+    #Find more DSL commands in rdocs under 'interface'
+    end
+end
+
+#This is the service class.
+#It declares all the resource interfaces that exist within the service application.
```

Comment:
More formatting - throughout the file can you change comments so they have a space between the `#` and the first word.
```suggestion
# This is the service class.
# It declares all the resource interfaces that exist within the service application.
```